### ISSUE TITLE
Selecting and drag/drop on touch devices with iOS10 (Safari).

### ISSUE THREAD

**Issue by sledgas** [fullcalendar/fullcalendar-scheduler#230]
Since the release of iOS10, what I believe is the `.fc-scroller` div in both agenda and timeline view scrolls along with user interactions, such as selecting and moving events. This makes it virtually impossible to draw a selection or rearrange events as the event container also moves.

For reproducing, I suggest visiting the demo on [https://fullcalendar.io/js/fullcalendar-3.0.1/demos/selectable.html](https://fullcalendar.io/js/fullcalendar-3.0.1/demos/selectable.html) on an iOS 10 device, and either switching to an agenda view (week or day) and try to draw a selection, or try to move an event in month view.

AFAIK, this worked in Safari on iOS9. My guess is that something has changed regarding `-webkit-overflow-scrolling: touch;`, or has to do with how Safari interpreters the new 3D Touch functions, although this is verified on a device without 3D Touch support.

**Comment by sledgas** [comment]
Quite a hack, but for now I'm solving it like this. Maybe something similar (and better) could be taken into consideration?

```
var isSelecting = false;
$('.fc-scroller').on('touchmove', function (event) {
    if (isSelecting) event.preventDefault();
});
```
```
$('#myDiv').fullCalendar({
    ...
    dragScroll: false,
    selectAllow: function (selectInfo) {
        isSelecting = true;
    },
    ...
});
```

**Comment by arshaw** [comment]
thanks, i'll upgrade to iOS10 and try to reproduce/fix this.

**Comment by cecrs** [comment]
Moving this comment from an already closed issue:

I can confirm this issue, and it can also be seen in the vertical resources demo here -
https://fullcalendar.io/js/fullcalendar-scheduler-1.5.0/demos/vertical-resource-view.html

This is also an issue in the timeline view.  I'm seeing the behavior on an iPhone 6, running ios 10.2. iPad mini in portrait mode, and everything bigger, works great for both vertical resource views and timelines. Interestingly enough, I can only reproduce this on an actual device, both the Safari Responsive design mode and the Chrome device inspector work, using the simulated iPhone 6.

Please let me know if you need any additional information!

**Comment by arshaw** [comment]
I can confirm this. Corresponding issue in the core project: https://github.com/fullcalendar/fullcalendar/issues/3403

**Comment by arshaw** [comment]
Fixed in [v1.5.1](https://github.com/fullcalendar/fullcalendar-scheduler/releases/tag/v1.5.1)
